controllers/k8ssandra/stargate.go

```diff
@@ -36,6 +36,11 @@ func (r *K8ssandraClusterReconciler) reconcileStargate(
 	actualStargate := &stargateapi.Stargate{}
 	logger = logger.WithValues("Stargate", stargateKey)
 
+	if actualDc.Spec.Stopped && stargateTemplate != nil {
```

Comment:
I've asked myself this question too. I believe a Stargate node can operate normally even if the local DC is down. It could be a good thing to keep it up and running.